op-e2e/actions/helpers/l1_miner.go

```diff
@@ -220,14 +221,20 @@ func (s *L1Miner) ActL1EndBlock(t Testing) *types.Block {
 		withdrawals = make([]*types.Withdrawal, 0)
 	}
 
-	block := types.NewBlock(s.l1BuildingHeader, &types.Body{Transactions: s.L1Transactions, Withdrawals: withdrawals}, s.l1Receipts, trie.NewStackTrie(nil), types.DefaultBlockConfig)
 	isCancun := s.l1Cfg.Config.IsCancun(s.l1BuildingHeader.Number, s.l1BuildingHeader.Time)
 	if isCancun {
-		parent := s.l1Chain.GetHeaderByHash(s.l1BuildingHeader.ParentHash)
-		excessBlobGas := eip4844.CalcExcessBlobGas(s.l1Cfg.Config, parent, s.l1BuildingHeader.Time)
-		s.l1BuildingHeader.ExcessBlobGas = &excessBlobGas
+		blobGasUsed := uint64(len(s.l1BuildingBlobSidecars)) * params.BlobTxBlobGasPerBlob
+		s.l1BuildingHeader.BlobGasUsed = &blobGasUsed
+	}
+
+	if s.l1Cfg.Config.IsPrague(s.l1BuildingHeader.Number, s.l1BuildingHeader.Time) {
+		// Don't process requests for now.
+		reqHash := types.CalcRequestsHash([][]byte{})
+		s.l1BuildingHeader.RequestsHash = &reqHash
 	}
```

Comment:
This was missing. Wondering why no other L1 Prague tests were failing? Maybe no other action tests use L1 Prague yet?!

Reply:
The one from [this PR](https://github.com/ethereum-optimism/optimism/pull/14006/files) does, but couldn't be merged until recently.